src/libraries/System.Collections/tests/Generic/PriorityQueue/PriorityQueue.Generic.Tests.cs

```diff
@@ -52,124 +50,100 @@ public void PriorityQueue_Generic_Constructor()
 
         [Theory]
         [MemberData(nameof(ValidCollectionSizes))]
-        public void PriorityQueue_Generic_Constructor_int(int initialCapacity)
+        public void PriorityQueue_EmptyCollection_UnorderedItemsIsEmpty(int initialCapacity)
         {
             var queue = new PriorityQueue<TElement, TPriority>(initialCapacity);
-
             Assert.Empty(queue.UnorderedItems);
         }
 
         [Fact]
-        public void PriorityQueue_Generic_Constructor_int_Negative_ThrowsArgumentOutOfRangeException()
-        {
-            AssertExtensions.Throws<ArgumentOutOfRangeException>("initialCapacity", () => new PriorityQueue<TElement, TPriority>(-1));
-            AssertExtensions.Throws<ArgumentOutOfRangeException>("initialCapacity", () => new PriorityQueue<TElement, TPriority>(int.MinValue));
-        }
-
-        [Fact]
-        public void PriorityQueue_Generic_Constructor_IComparer()
+        public void PriorityQueue_ComparerConstructor_ComparerShouldEqualParameter()
         {
-            IComparer<TPriority> comparer = Comparer<TPriority>.Default;
+            IComparer<TPriority> comparer = GetIComparer();
             var queue = new PriorityQueue<TElement, TPriority>(comparer);
-
             Assert.Equal(comparer, queue.Comparer);
         }
 
         [Fact]
-        public void PriorityQueue_Generic_Constructor_IComparer_Null()
+        public void PriorityQueue_ComparerConstructorNull_ComparerShouldEqualDefaultComparer()
         {
-            var queue = new PriorityQueue<TElement, TPriority>((IComparer<TPriority>)null);
+            var queue = new PriorityQueue<TElement, TPriority>(comparer: null);
+            Assert.Equal(0, queue.Count);
             Assert.Equal(Comparer<TPriority>.Default, queue.Comparer);
         }
 
         [Theory]
         [MemberData(nameof(ValidCollectionSizes))]
-        public void PriorityQueue_Generic_Constructor_int_IComparer(int initialCapacity)
+        public void PriorityQueue_CapacityConstructor_ComparerShouldEqualDefaultComparer(int initialCapacity)
         {
-            IComparer<TPriority> comparer = Comparer<TPriority>.Default;
             var queue = new PriorityQueue<TElement, TPriority>(initialCapacity);
-
             Assert.Empty(queue.UnorderedItems);
-            Assert.Equal(comparer, queue.Comparer);
+            Assert.Equal(Comparer<TPriority>.Default, queue.Comparer);
```

Comment:
It doesn't really matter, but consider using Assert.Same when you're looking to validate reference equality.